.github/workflows/lint.yaml

```diff
@@ -29,10 +29,17 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       
-      - name: Use shellcheck
-        run: sudo apt-get install shellcheck
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.16'
+
+      - name: install dependencies
+        run: |
+          sudo apt-get install shellcheck
+          GO111MODULE=on go get mvdan.cc/sh/v3/cmd/shfmt
       
       - name: Run shellcheck
         run:  |
           pwd
-          shellcheck $(find . -name "*.sh")
+          shfmt -f "$(git rev-parse --show-toplevel)" | xargs shellcheck
```

Comment:
You will probably have to revert it to `$(find . -name "*.sh")` since shfmt checks the shebang and includes `jdtls`

Reply:
I don't believe that's a good solution. I can disable the rule for shellcheck, but the whole point of it is to complain about potential issues. I can at least replace `which` with `command -v`.

Reply:
I'm not the one you need to convince ;)